e2e/nomostest/git-server.go

```diff
@@ -103,6 +103,16 @@ func isAvailableDeployment(o client.Object) error {
 		return fmt.Errorf("%w for deployment/%s in namespace %s: got %d available replicas, want %d\n\n%s",
 			ErrFailedPredicate, d.GetName(), d.GetNamespace(), available, want, string(jsn))
 	}
+	unavailable := d.Status.UnavailableReplicas
```

Comment:
Can we just replace this whole isAvailableDeployment with kstatus.Compute? That's what I did in the reconciler-manager recently with `computeDeploymentStatus`.

Reply:
DONE